static/assets/css/stylesheet.css

```diff
@@ -89,6 +89,16 @@ video {
   font: inherit;
   vertical-align: baseline;
 }
+video {
```

Comment:
This seems a bit general. Maybe make it more specific -- give it a css class or something?

Reply:
Done!